src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -5199,6 +5199,36 @@ private BoundExpression BindCollectionExpression(CollectionExpressionSyntax synt
 
             MessageID.IDS_FeatureCollectionExpressions.CheckFeatureAvailability(diagnostics, syntax, syntax.OpenBracketToken.GetLocation());
 
+            for (int i = 1, n = syntax.Elements.Count - 1; i < n; i++)
```

Comment:
> - 1 [](http://example.com/codeflow?start=53&length=4)

Is this correct?

Reply:
yes.  we want to allow with elemnets as the first element, and fail on every subsequent element.  this can be doc'ed if you want.

Reply:
Yes, but it looks like we're only checking up to `Count - 2`.

Reply:
whoopsie!